db/migrate.go

```diff
@@ -0,0 +1,50 @@
+package db
+
+import (
+	"embed"
+	"fmt"
+	"net/http"
+
+	"github.com/golang-migrate/migrate/v4"
+	_ "github.com/golang-migrate/migrate/v4/database"
+	_ "github.com/golang-migrate/migrate/v4/database/mysql"
+	_ "github.com/golang-migrate/migrate/v4/database/postgres"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+	"github.com/golang-migrate/migrate/v4/source/httpfs"
+)
+
+func RunMigrations(config Config, embeddedMigrations embed.FS, resourcePath string) error {
```

Comment:
Let's use the interface `fs.FS` instead of the implementation `embed.FS`.

Same for `RunRollback` method.

Reply:
fixed in 38d9c35a2656802687a629b8052023cb0520d5f2